fiftyone/utils/torch.py

```
@@ -2605,6 +2667,245 @@ def _parse_inputs(
         return image_paths, sample_ids, patch_edges, patches
 
 
+class TorchVideoFramesIterableDataset(IterableDataset):
+    def __init__(
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)